extensions/adobe/experience/decisioning/frequency-capping-constraints.schema.json

```diff
@@ -51,49 +51,57 @@
                   "placement": "To comply with offer placement."
                 }
               },
-              "xdm:placement": {
```

Comment:
@jshaik-adobe Can you please confirm that there is no data set uploaded in platform associated to this field group referenced in any schema ?

Reply:
@prabhum2 , we don't have any data for deleted fields

Reply:
@jshaik-adobe , to be on the safer side, can we just deprecate these properties ? We generally do not delete properties from a schema which is marked as "stable".

Reply:
@mjasrotia deprecated unused properties